packages/workspace/src/workspace/workspace.ts

```diff
@@ -219,7 +219,12 @@ export const loadWorkspace = async (
     makeArray(workspaceConfig.envs).find(e => e.name === currentEnv()) as EnvConfig
   const currentEnvsConf = (): EnvConfig[] =>
     workspaceConfig.envs
-  const services = (): string[] => makeArray(currentEnvConf().services)
+  const services = (env?: string): string[] => {
+    const envConf = env
+      ? makeArray(workspaceConfig.envs).find(e => e.name === currentEnv()) as EnvConfig
+      : currentEnvConf()
+    return envConf ? makeArray(envConf.services) : []
```

Comment:
nit:
```suggestion
    return makeArray(envConf?.services)
```
though you have casted both options `as EnvConfig`, so according to that, `envConf` cannot be undefined and this should actually be
```suggestion
    return makeArray(envConf.services)
```